save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/ContainerLogController.kt

```diff
@@ -0,0 +1,78 @@
+package com.saveourtool.save.backend.controllers
+
+import com.saveourtool.save.backend.service.AgentService
+import com.saveourtool.save.configs.ApiSwaggerSupport
+import com.saveourtool.save.service.LogService
+import com.saveourtool.save.utils.blockingToMono
+import com.saveourtool.save.utils.toInstantAtDefaultZone
+import com.saveourtool.save.v1
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.Parameter
+import io.swagger.v3.oas.annotations.Parameters
+import io.swagger.v3.oas.annotations.enums.ParameterIn
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import io.swagger.v3.oas.annotations.tags.Tags
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import reactor.core.publisher.Mono
+import java.time.Instant
+
+/**
+ * Controller to fetch logs
+ */
+@ApiSwaggerSupport
+@Tags(
+    Tag(name = "logs"),
+)
+@RestController
+@RequestMapping(path = ["/api/$v1/logs"])
+class ContainerLogController(
+    private val logService: LogService,
+    private val agentService: AgentService,
+) {
+    @GetMapping("/by-container-name")
+    @Operation(
+        method = "GET",
+        summary = "Get logs for containerName in provided time range.",
+        description = "Get logs for containerName in provided time range.",
+    )
+    @Parameters(
+        Parameter(name = "containerName", `in` = ParameterIn.QUERY, description = "name of a container", required = true),
+        Parameter(name = "from", `in` = ParameterIn.QUERY, description = "start of requested time range", required = true),
```

Comment:
This was added for testing purposes, I'm not sure that such endpoint should be public api (or any api) in our service. There is Grafana for it.
It will be removed after testing on real env and implementing log for CPG

Reply:
decided to leave it for now (about endpoint)

specified format explicitly and updated description

I use everywhere default time zone, because I think we can control it by parameter for java machine